tensorboard/webapp/feature_flag/types.ts

```diff
@@ -44,8 +44,8 @@ export interface FeatureFlags {
   forceSvg: boolean;
   // Whether to enable the "sticky" data table in scalar cards.
   enabledScalarDataTable: boolean;
-  // If enabled causes the feature flags modal to appear.
-  enableShowFlags: boolean;
+  // If defined causes the feature flags modal to appear.
+  showFlags: string | undefined;
```

Comment:
Because this flag is not overridden in any of the other tensorboard versions this rename should be fine